assets/js/vue/components/stats/DatasetSizeRanges.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+    <DxChart
+        id="dsr-chart"
+        data-source="/stats/data/dataset-size-ranges"
```

Comment:
We could add Routing.generate here i guess, and remove the hard coded routes

Reply:
How would you implements that?

Reply:
Use a data variable here, and define the data variable as 
`data() {
        return {
            datasetSizeUrl: Routing.generate("pelagos_app_ui_stats_getdatasetsizeranges),
        };
    }, `